### ISSUE TITLE
Defaults speed limits are overwritten by RViz config

### ISSUE THREAD

**Issue by NFiedler** [ros-planning/moveit#2906]
Instead of using the configured default speed limits from the joint_limits.yaml, the RViz plugin loads them from the RViz config file. Thus, if the defaults change, existing RViz configs are not affected. This can result in confusion as the default changed from 1.0 to 0.1.

https://github.com/ros-planning/moveit/blob/d57721cf8503fe893347196cc1e1f9249ec57777/moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp#L1332-L1335

**Comment by simonschmeisser** [comment]
Those should be applied as a factor to the speed limits (per joint) configured in `joint_limits.yaml`, is that not the case?

**Comment by v4hn** [comment]
> Those should be applied as a factor to the speed limits (per joint) configured in `joint_limits.yaml`, is that not the case?

Yes, they are. But the default values for these factors can be set in the same file as well.
https://github.com/ros-planning/moveit/blob/master/MIGRATION.md?plain=1#L13-L17

Changing them in the joint_limits.yaml will not effect existing rviz configurations though.

I see two alternative ways to improve this:
- do not load the parameters from rviz configs at all, but use the default from the parameter server
- or in addition to the currently saved setting also store their current default value in the ros parameters in the rviz config, so we can detect changed defaults and overrule the old saved setting.
